$OpenBSD: patch-tmac_doc_tmac,v 1.2 2013/03/30 23:10:05 schwarze Exp $
Always format .Pa the same way, as requested by jmc@ (local change).
--- tmac/doc.tmac.orig	Sun Dec 30 08:40:29 2012
+++ tmac/doc.tmac	Tue Jan  1 17:37:17 2013
@@ -3759,8 +3759,6 @@
 .
 .        if \n[doc-in-files-section] \{\
 .          ds doc-saved-Pa-font "\*[doc-Pa-font]
-.          if n \
-.            ds doc-Pa-font "\*[doc-No-font]
 .        \}
 .
 .        ie (\n[doc-type1] == 1) \
